tree.go

```diff
@@ -62,21 +62,34 @@ func (n *Tree[K, V]) IsEmpty() bool {
 	return n == nil || n.size == 0
 }
 
-// Find returns the pair associated with key in the tree. Will return an empty pair if no such item exists.
-func (n *Tree[K, V]) Find(key K) Pair[K, V] {
+// Contains returns true if the tree contains the key.
+func (n *Tree[K, V]) Contains(key K) bool {
+	_, found := n.FindOpt(key)
+	return found
+}
+
```

Comment:
Please adjust the comment to also indicate it returns the zero value if the key does not exist in the tree.